EMIP/1-1000/EMIP00816.xml

```diff
@@ -29,34 +29,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <collection>EMIP</collection>
                   <idno facs="EMIP/Codices/816/">Chelekot Silasie 75</idno>
                </msIdentifier>
+
                <msContents>
-                  <summary>Homilies on Right Faith, ድርሳነ፡ ርቱዓ፡ ሃይማኖት።; Homily on Wälätä Maryam by Hǝrǝyaqos, ድርሳነ፡ ወለተ፡ ማርያም፡</summary>
-                  <msItem xml:id="ms_i1">
-                     <textLang mainLang="gez">Gǝ‘ǝz</textLang>
-                  </msItem>
+                  <summary>Homilies on Right Faith, ድርሳነ፡ ርቱዓ፡ ሃይማኖት።; Homily on Walatta Māryām by Hǝrǝyāqos, ድርሳነ፡ ወለተ፡ ማርያም፡</summary>
+
```

Comment:
Transliteration should be in line with the guidelines, https://betamasaheft.eu/Guidelines/?q=transliteration&id=transliteration-principles note in particular the first form vowel ä

Reply:
In my Atom the transliteration is correct i. e. Walatta Māryām and Hǝrǝyāqos.

Reply:
My mistake! Apologieṣ́